packages/primitives/src/styled.ts

```diff
@@ -20,23 +22,22 @@ function getShouldForwardProp(component /*: React.ElementType */) {
 /**
  * a function that returns a styled component which render styles on multiple targets with same code
  */
-
-/*
-type CreateStyledComponent = (
-  ...styles: any
-) => React.StatelessFunctionalComponent<any> & {
-  withComponent: (component: any) => React.StatelessFunctionalComponent<any>
+type CreateStyledComponent = (...styles: any) => React.FC<any> & {
+  withComponent: (component: any) => React.FC<any>
 }
 
-type BaseStyled = (tag: React.ElementType) => CreateStyledComponent
+/** @internal */
+export type BaseStyled = (
```

Comment:
out of curiosity - what kind of an effect the internal annotation might have?

Reply:
Does not have any functional effect, just wanted to document that this type should not be exported from the package's index file.